Dwallace/1154 remove ContentExpression language restriction (#1292)
semanticarts/gist: docs/release_notes/issue1154-remove-language-restriction-from-ContentExpression.md

### Patch Updates

> - Removed Language restriction from `gist:ContentExpression` and updated the definition to say that if there is text, it "will" be in a language rather than "may" be. Issue [#1154](https://github.com/semanticarts/gist/issues/1154).

Reviewer: ```suggestion
### Major Updates

- Removed restriction from `gist:ContentExpression` and updated the definition. Issue [#1154](https://github.com/semanticarts/gist/issues/1154).
```
